harmonica/forward/prism_layer.py

```diff
@@ -469,3 +486,43 @@ def to_pyvista(self):
                 for data_var in self._obj.data_vars
             }
         return prism_to_pyvista(prisms, properties=properties)
+
+
+def _discard_thin_prisms(
+    prisms,
+    density,
+    thickness_threshold,
+):
+    """
+    Discard prisms with a thickness below a threshold
+
+    Parameters
+    ----------
+    prisms : 2d-array
+        Array containing the boundaries of the prisms in the following order:
+        ``w``, ``e``, ``s``, ``n``, ``bottom``, ``top``.
+        The array must have the following shape: (``n_prisms``, 6), where
+        ``n_prisms`` is the total number of prisms.
+    density : 1d-array
+        Array containing the density of each prism in kg/m^3. Must have the
+        same size as the number of prisms.
+    thickness_threshold : int or float
+        Minumum prism thickness needed to be to including in the forward
+        gravity calculation. Default to 0.
+
+    Returns
+    -------
+    prisms : 2d-array
+        A copy of the ``prisms`` array that doesn't include the thin prisms.
+    density : 1d-array
+        A copy of the ``density`` array that doesn't include the density values
+        for thin prisms).
+    """
+    west, east, south, north, bottom, top = tuple(prisms[:, i] for i in range(6))
+    # Mark prisms with thickness <= threshold  as null prisms
+    thickness = abs(top - bottom)
```

Comment:
I think this private function should assume that `top >= bottom` so the `abs` shouldn't be needed. If for some reason a prism in the layer has `bottom < top` we have a bug and we need to catch it beforehand on a specific check.
```suggestion
    thickness = top - bottom
```

Reply:
Ok this makes sense. I was thinking that sense users can provide a `reference` which is actually above `surface` I would take the absolute value, but I didn't realize that harmonica would have flipped them internally. I think I'll open a separate issue to discuss this flipping of `top` and `bottom` when `bottom` is above `top`.